PhysicsTools/PyTorch/test/testPipelineStandalone.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+ 
+function die { echo Failed $1: status $2 ; exit $2 ; }
+
+SCRIPT="PhysicsTools/PyTorch/test/testPipeline.py"
```

Comment:
Is there some other reason to have this `testPipelineStandalone.sh` than avoiding the `${LOCALTOP}/src` part in the `testPipeline.sh`?